### ISSUE TITLE
Lightning Network or "LN" and how to choose

### ISSUE THREAD

**Issue by aantonop** [lnbook/lnbook#101]
I want to avoid writing "Lightning Network" a thousand times in this book. So I want to use the acronym "LN", as a shortcut. In many places this works well. 

For example "a LN node" or even (sic) "an LN node". Similarly, wherever we use Lightning Network as a _qualifier_ to another noun such as node, wallet, transaction, channel etc.

However, in practice, I have found it difficult to read and write the LN acronym if we are not using it as a qualifier. For example the sentence:

"The Lightning Network is a layer-2 network that operates..." 

In that kind of sentence, where the Lightning Network is the subject of the sentence and not a qualifier, it feels weird to write and difficult to read as an acronym:

"The LN is a layer-2 network that operates..." 

I propose that we use the acronym "LN" where it is used as a qualifier (LN node, LN transaction, LN wallet) and write "The Lightning Network" in full where it is the subject of the sentence and not a qualifier.  

Thoughts?

**Comment by bolatovumar** [comment]
This sounds reasonable to me.

Would it make sense to introduce the term in the beginning and state that the abbreviated form will be used in the rest of the book? Obviously this doesn't mean that ONLY the abbreviated form can be used after the term is initially introduced.

**Comment by aantonop** [comment]
Yes the acronym is introduced in the first paragraph of chapter 1. I just
felt like it was too clunky to use everywhere.

**Comment by inpharmaticist** [comment]
I've been seeing "LNP/BP" on twitter to describe the use of Lightning on top of Bitcoin. Is there an accepted difference between LNP and LN? 

Personally, I think when used as the subject of the sentence, I'd think of it as:

"Lightning is a layer-2 network that operates..."
as opposed to 
"The LN is a layer-2 network that operates..."

I think that treats it as more of a proper name or trademark. It's analogous to saying either "the Bitcoin network operates..." interchangeably with "Bitcoin operates..."

**Comment by bolatovumar** [comment]
I read through the first chapter and "The LN" does feel clunky to read so this suggestion definitely improves the readability.

**Comment by aantonop** [comment]
Based on this feedback, here is the final style decision:

Where "Lightning Network" is used as a qualifier, it is shortened to LN, such as in LN wallet, LN transaction, LN node. 

Where "Lightning Network" is used as a proper noun, prefixed by "the", it is written in full and capitalized as "the Lightning Network".

**Comment by DariusParvin** [comment]
There's a third option which I don't think was considered above, which is using just "Lightning" when used as a qualifier, e.g. "Lightning node", "Lightning wallet", "Lightning transaction". 

I noticed "Lightning wallet" was written a lot on chapter 2. To me this feels the most natural and is also what most people say in real life. With "LN wallet" I tend to either read it as "ell en" wallet, or mentally convert it to "lightning network wallet", both of which are a bit awkward.

It's also consistent with how we describe Bitcoin nodes/wallets etc. For example, we don't say "Bitcoin Network Wallet" or "Bitcoin Blockchain Transaction".

**Comment by aantonop** [comment]
After some review, we ended up with an additional problem, which is the difference of opinion about how to write "a Lightning Network wallet" if we use the LN contraction. 

I wrote it as "an LN wallet", because I read it as "an el-en wallet", but others changed it to the (more grammatically correct) "a LN wallet" to read a "a Lightning Network wallet". 

@DariusParvin suggestion above as a "third option" is far better I think, so I'm revising!

I am switching it to simply "a Lightning wallet" and "a Lightning node", which resolves the problem of how it is pronounced/read and avoids the overly long "Lightning Network". Also, it is consistent with popular use of the term "Lightning" on its own.
